service/controller/clusterapi/v27/resource/vpccidr/delete.go

```diff
@@ -2,22 +2,8 @@ package vpccidr
 
 import (
 	"context"
-
-	"github.com/giantswarm/microerror"
-
-	"github.com/giantswarm/aws-operator/service/controller/clusterapi/v27/key"
 )
 
 func (r *Resource) EnsureDeleted(ctx context.Context, obj interface{}) error {
-	cluster, err := key.ToCluster(obj)
-	if err != nil {
-		return microerror.Mask(err)
-	}
-
-	err = r.addVPCCIDRToContext(ctx, cluster)
-	if err != nil {
-		return microerror.Mask(err)
-	}
```

Comment:
We also do not need to add the CIDR to the context when we delete a cluster. That was an unnecessary step taken so far.